docs/forms.md

```diff
@@ -0,0 +1,143 @@
+# AcroForms in PDFKit
+
+AcroForms are interactive features of the PDF format, and they make it possible
+to include things like text fields, buttons and actions. To include AcroForms
+you must call the document `initAcroForm()` method.
+
+* `initAcroform()` - Must be called when using AcroForms
+
+## AcroForm Methods
+
+AcroForm elements are _Widget Annotations_ and are added using the
+`widgetAnnotation` method. Additional methods listed below are shortcut methods
+that call the `widgetAnnotation` method. The list of the available _Widget
+Annotation_ document methods is:
+
+* `widgetAnnotation( name, x, y, width, height, options)`
+* `formText( name, x, y, width, height, options)`
+* `formPushButton( name, x, y, width, height, name, options)`
+* `formRadioButton( name, x, y, width, height, options)`
+* `formNoToggleToOffButton( name, x, y, width, height, options)`
+* `formChoice( name, x, y, width, height, options)`
+
+### Options Parameter
+
+Some Widget Annotations have a `color` option that can be specified. You can use
+an array of RGB values, a hex color, or a named CSS color value for that option.
+
+* `backgroundColor` - button background color
+* `borderColor` - button border color
+
+Other `options` conveniences include:
+
+* `label` - set button text labels (will set <<MK <<CA (label)>> >>)
+* `align` - set to `left`, `center` or `right` for text within the
+  Widget Annotation
+* Field flags that will set bits in `Ff`
+  * `readyOnly`: 1,
+  * `required`: 2,
+  * `noExport`: 4,
+  * `multiline`: 0x1000,
+  * `password`: 0x2000,
+  * `toggleToOffButton`: 0x4000,
+  * `radioButton`: 0x8000, (will be set when calling the `formRadioButton` method)
+  * `pushButton`: 0x10000 (will be set when calling the `formPushButton` method)
+  * `toggleToOffButton`: 0x10000 (will be set when calling the `formNoToggleToOffButton` method)
+  * `combo`: 0x20000,
+  * `edit`: 0x40000,
+  * `sort`: 0x80000
+
+When using the `formChoice` method, set `options.Opt` to the array of choices. 
```

Comment:
Might be nice to spell this out a bit more. What are the choices supposed to look like? Just an array of strings or some kind of object?

Also, could we name this field `options.choices` or something rather than the shorthand directly from the PDF spec?